From bc27df07cfb02caf83ab153c98a8106934109ac2 Mon Sep 17 00:00:00 2001
From: Andrew kim <andrew.kim@intel.com>
Date: Wed, 22 May 2019 12:25:38 -0700
Subject: [PATCH 2320/2345] x86/speculation/rdt_pseudo_lock: cpu name change

changed the platform name to SoC name that have been
validated to support pseudo_locking.

Signed-off-by: Andrew kim <andrew.kim@intel.com>
---
 arch/x86/kernel/cpu/intel_rdt_pseudo_lock.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/x86/kernel/cpu/intel_rdt_pseudo_lock.c b/arch/x86/kernel/cpu/intel_rdt_pseudo_lock.c
index 7b358d9..07fb583 100644
--- a/arch/x86/kernel/cpu/intel_rdt_pseudo_lock.c
+++ b/arch/x86/kernel/cpu/intel_rdt_pseudo_lock.c
@@ -93,7 +93,7 @@ static u64 get_prefetch_disable_bits(void)
 		 */
 		return 0xF;
 	case INTEL_FAM6_ATOM_GOLDMONT:
-	case INTEL_FAM6_ATOM_GEMINI_LAKE:
+	case INTEL_FAM6_ATOM_GOLDMONT_PLUS:
 		/*
 		 * SDM defines bits of MSR_MISC_FEATURE_CONTROL register
 		 * as:
@@ -1070,7 +1070,7 @@ static int measure_l2_residency(void *_plr)
 	 */
 	switch (boot_cpu_data.x86_model) {
 	case INTEL_FAM6_ATOM_GOLDMONT:
-	case INTEL_FAM6_ATOM_GEMINI_LAKE:
+	case INTEL_FAM6_ATOM_GOLDMONT_PLUS:
 		perf_miss_attr.config = X86_CONFIG(.event = 0xd1,
 						   .umask = 0x10);
 		perf_hit_attr.config = X86_CONFIG(.event = 0xd1,
-- 
2.7.4

